src/libs/fileDownload/FileUtils.js

```diff
@@ -154,6 +154,7 @@ const readFileAsync = (path, fileName) =>
             .then((blob) => {
                 const file = new File([blob], cleanFileName(fileName));
                 file.source = path;
+                file.uri = path;
```

Comment:
It does not look that we are using the `uri` anywhere, any reason we are adding it?

Reply:
I'll add a comment. On iOS, File objects do not seem to have a `uri` property, so uploading to the backend fails.